dev/reference/error-codes.md

```diff
@@ -19,6 +19,94 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
+| 8026 | The interface has no implementation. |
+| 8027 | The table schema version is outdated. |
+| 8028 | The table schema has changed. |
+| 8029 | Incorrect value. |
+| 8030 | After an unsigned positive integer is converted to a signed integer, it exceeds the maximum value and displays as a negative integer.|
+| 8031 | When being converted to an unsigned integer, a negative integer is converted to a positive integer. |
+| 8032 | Illegal `year` format. |
+| 8033 | Illegal `year` value. |
+| 8034 | Incorrect `datetime` value. |
+| 8036 | Illegal `time` format. |
+| 8037 | Illegal `week` format. |
+| 8038 | The field cannot obtain a default value. |
+| 8039 | Index offset is out of range. |
+| 8042 | The state of table schema does not exist. |
+| 8043 | The state of column information does not exist. |
+| 8044 | The state of index does not exist. |
+| 8045 | Illegal table data. |
+| 8046 | The state of column information is invisible. |
+| 8047 | The value of the system variable is not supported. This error usually occurs in the alarm information when the user sets a variable value that is not supported in the database.
+| 8048 | An unsupported database isolation level is set. |
+| 8049 | It fails to load the privilege related table. |
+| 8050 | An unsupported privilege type is set. |
+| 8051 | Unknown field type. |
+| 8052 | The serial number of the data packet from the client is incorrect. |
+| 8053 | An illegal auto-incrementing column value is obtained. |
+| 8055 | The current snapshot is too old. The data may have been garbage collected. |
+| 8056 | Illegal table ID. |
+| 8057 | Illegal field type. |
+| 8058 | You apply an automatic variable type that does not exist. |
+| 8059 | It fails to obtain an auto-random ID. |
+| 8060 | Illegal auto-incrementing offset. |
+| 8061 | Unsupported SQL Hint. |
+| 8062 | An illegal token is used in SQL Hint. It conflicts with reserved words in SQL Hint. |
+| 8063 | The limited memory usage set in SQL Hint exceeds the upper limit of the system. The setting in SQL Hint is ignored. |
+| 8064 | It fails to parse SQL Hint. |
+| 8065 | An illegal integer is used in SQL Hint. |
+| 8066 | The second parameter in the `JSON_OBJECTAGG` function is illegal. |
+| 8101 | The format of plugin ID is incorrect. | The correct format is `[name]-[version]`, and no `-` is allowed in `name` and `version`. |
+| 8102 | Unable to read the plugin definition information. |
+| 8103 | The plugin name is incorrect. |
+| 8104 | The plugin version does not match. |
+| 8105 | The plugin is repeatedly loaded. |
+| 8106 | The plugin defines a system variable whose name does not begin with the plugin name. |
+| 8107 | The loaded plugin does not specify a version, or the specified version is too low. |
+| 8108 | Unsupported execution plan type. |
+| 8109 | The specified index cannot be found when the index is analyzed. |
+| 8110 | The Cartesian product operation cannot be executed. | Set `cross-join` in the configuration to `true`. |
```

Comment:
```suggestion
| 8110 | The Cartesian product operation cannot be executed. Set `cross-join` in the configuration to `true`. |
```

Reply:
The English table schema has a `Solution` column. Should we put solutions in that column?

Reply:
ok